### DISCUSSION TITLE
Sessions Table: Scores Column

### DISCUSSION THREAD

**Discussion by jannikmaierhoefer** [langfuse/langfuse#4120] [Ideas]
### Describe the feature or potential improvement

High-level averages and sums of trace scores in the session table view.

**Comment by GorjanV8** [comment]
Definitely a use-case my team came across. Seems like there's a similar discussion at #2728. Aggregated scores is incredibly common and the primary desired result of any evaluation.

**Comment by milagro-multion** [comment]
It would be really helpful to have this feature available.

In our use case, it would be good to also support custom string metrics. For example, we sometimes summarize the of most common errors in traces.
